tests/test.py

```diff
@@ -135,22 +137,43 @@ def test_bundle_api(self):
             requests.codes.ok)
 
 class TestSyncUtils(unittest.TestCase):
-    def test_sync_blob(self):
-        gcs_bucket_name, s3_bucket_name = os.environ["DSS_GCS_TEST_BUCKET"], os.environ["DSS_S3_TEST_BUCKET"]
-        logger = logging.getLogger(__name__)
-        s3 = boto3.resource("s3")
-        payload = os.urandom(2**20)
-        test_key = "hca-dss-s3-to-gcs-sync-test"
-        s3.Bucket(s3_bucket_name).Object(test_key).put(Body=payload)
-        sync.sync_blob(source_platform="s3", source_key=test_key, dest_platform="gcs", logger=logger)
-        # TODO: wait for GCSTS job and read back key
-
-        test_key = "hca-dss-gcs-to-s3-sync-test"
+    def setUp(self):
+        self.gcs_bucket_name = os.environ["DSS_GCS_TEST_BUCKET"]
+        self.s3_bucket_name = os.environ["DSS_S3_TEST_BUCKET"]
+        self.logger = logging.getLogger(__name__)
         gcs_key_file = os.environ["GOOGLE_APPLICATION_CREDENTIALS"]
         gcs = google.cloud.storage.Client.from_service_account_json(gcs_key_file)
-        gcs.bucket(gcs_bucket_name).blob(test_key).upload_from_string(payload)
-        sync.sync_blob(source_platform="gcs", source_key=test_key, dest_platform="s3", logger=logger)
-        dest_blob = s3.Bucket(s3_bucket_name).Object(test_key)
+        self.gcs_bucket = gcs.bucket(self.gcs_bucket_name)
+        s3 = boto3.resource("s3")
+        self.s3_bucket = s3.Bucket(self.s3_bucket_name)
+
+    def cleanup_sync_test_objects(self, prefix="hca-dss-sync-test", age=datetime.timedelta(days=1)):
+        for key in self.s3_bucket.objects.filter(Prefix=prefix):
```

Comment:
This is a great suggestion but since I'm in a hurry to get stuff fixed, I'm going to merge this now and fix later. Filed #111 to track.